src/pudl/etl/glue_assets.py

```diff
@@ -44,22 +46,22 @@ def create_glue_tables(
 
     # Add the EPA to EIA crosswalk, but only if the eia data is being processed.
     # Otherwise the foreign key references will have nothing to point at:
-    ds = context.resources.datastore
-    if dataset_settings.glue.eia:
-        # Check to see whether the settings file indicates the processing of all
-        # available EIA years.
-        processing_all_eia_years = (
-            dataset_settings.eia.eia860.years
-            == dataset_settings.eia.eia860.data_source.working_partitions["years"]
-        )
-        glue_raw_dfs = pudl.glue.epacamd_eia.extract(ds)
-        glue_transformed_dfs = pudl.glue.epacamd_eia.transform(
-            glue_raw_dfs,
-            generators_entity_eia,
-            boilers_entity_eia,
-            processing_all_eia_years,
-        )
-        glue_dfs.update(glue_transformed_dfs)
+    # ds = context.resources.datastore
+    # if dataset_settings.glue.eia:
+    #     # Check to see whether the settings file indicates the processing of all
+    #     # available EIA years.
+    #     processing_all_eia_years = (
+    #         dataset_settings.eia.eia860.years
+    #         == dataset_settings.eia.eia860.data_source.working_partitions["years"]
+    #     )
+    #     glue_raw_dfs = pudl.glue.epacamd_eia.extract(ds)
+    #     glue_transformed_dfs = pudl.glue.epacamd_eia.transform(
+    #         glue_raw_dfs,
+    #         generators_entity_eia,
+    #         boilers_entity_eia,
+    #         processing_all_eia_years,
+    #     )
+    #     glue_dfs.update(glue_transformed_dfs)
```

Comment:
Should this be deleted?

Reply:
Also, I'm not seeing the `epacamd_eia` asset in the DAG anymore. Is it now called `clean_epacamd_eia`? If so, does the metadata need to be updated?